flightdata-kafka-producer/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM maven:3.5.3-jdk-8-slim as BUILD
+WORKDIR /build
+COPY ./pom.xml /build/pom.xml
+COPY ./src/ /build/src/
+RUN mvn -f /build/pom.xml clean package
+
+FROM anapsix/alpine-java:latest
+WORKDIR /workdir/
+RUN apk update && apk add --update netcat-openbsd
+COPY --from=BUILD /build/target/flightdata-kafka-producer-1.0-jar-with-dependencies.jar .
+ENTRYPOINT while( ! nc -z -v -w5 kafka 9092 ); do sleep 5; echo "waiting for kafka"; done && \
```

Comment:
Got it done in newest commit. Had some troubles since docker errors aren't the clearest. I was getting error:
```
standard_init_linux.go:190: exec user process caused "no such file or directory"
```
as it turned out, it was caused by CRLF line endings in `docker-entrypoint.sh` insted of linux LF ones.